enrolments/tests/views/test_session_clases.py

```diff
@@ -0,0 +1,72 @@
+from django.urls import reverse
```

Comment:
can we rename the file to test_session_classes.py 😄

Reply:
S tier speller